pkg/middleware/redirect_to_https.go

```diff
@@ -37,6 +37,7 @@ func redirectToHTTPS(httpsPort string, next http.Handler) http.Handler {
 		targetURL, _ := url.Parse(req.URL.String())
 		// Set the scheme to HTTPS
 		targetURL.Scheme = httpsScheme
+		targetURL.Host = req.Host
```

Comment:
Perhaps we should wrap this in a condition, if the request URL is empty then override it? Or do you think the host property should take precedence? If so, a comment as to why would be good

Reply:
Sure, I will wrap it in a condition.